### DISCUSSION TITLE
Picture is not displayed

### DISCUSSION THREAD

**Discussion by IvanIsak2000** [fastapi/fastapi#9458] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from fastapi.responses import HTMLResponse
from fastapi.responses import FileResponse


app = FastAPI()


@app.get("/", response_class=HTMLResponse)
async def read_items():
	html_content = """
	<html>


<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Dela+Gothic+One&family=Oswald&family=Ruslan+Display&display=swap" rel="stylesheet">

<style type="text/css">

body{
background-image: url("static/logo1.png");
background-size:100%
}


a:link {
color: f6f8f7; /* Цвет ссылок */
font-size: 30px
}

a:visited {
    color: rgb(255, 255, 255); /* Цвет посещенных ссылок */
   }

a:active {
color:rgb(100, 197, 170) ; /* Цвет активной ссылки */
font-size: 35px
}

li { 
  position: relative; 
}

a {
  color: #fff;
  text-transform: uppercase;
  text-decoration: none;
  letter-spacing: 0.15em;
  display: inline-block;
  padding: 15px 20px;
  position: relative;
}

a::after { 
  content: "";
  position: absolute;
  bottom: 0;
  left: 50%;
  display: block;
  background: none repeat scroll 0 0 transparent;
  height: 2px;
  width: 0;
  background:rgb(100, 197, 170);
  transition: width 0.3s ease 0s, left 0.3s ease 0s;
}

a:hover::after { 
  width: 100%; 
  left: 0; 
}

div { float: left; /*Задаем обтекание*/
	line-height: 0px;/*Высота строки + верт. центрирования текста*/ 
	/* font-size:;  */
	left: 0px;
	top: 50px;
	margin-top: 10px; 
	width: 350px; /*Фиксируем ширину блока*/ 
	margin-right: 10px; text-align: center; /*Центрируем текст по горизонтали*/ }






.name{
font-family: 'Dela Gothic One', cursive;
font-family: 'Oswald', sans-serif;
color: rgb(100, 197, 170);
font-size: 250px;
padding: 10;
/* margin-left: 100px; */
position: relative;
top: 170px; 
left: 30px;


}



</style>
<head>
 	<title>The Commonwealth</title>

</head>

<body>
</div >
	<div id = 'line_block' ; class="link-ease-in-out">
		<center>
			
			<p class="a" ><a href="https://vk.com/thecommonwealth2020" style="text-decoration: none" title="">VKontakte</a></p>
		</center>
	</div>

	<div id = 'line_block' ;class="link-ease-in-out">
		<center>
			<p class="a" ><a href="https://vk.com/away.php?to=https%3A%2F%2Ft.me%2Fthecommonwealth2022" style="text-decoration: none" title="">Telegram</a></p>
		</center>
	</div>

	<div id = 'line_block' ; class="link-ease-in-out">
		<center>
			<p class="a" ><a href="https://vk.com/doc403210316_657773170?hash=wL0PIuCs5GU5CWq1g2cpODL01ziPSK0aZ8wXuO3jCmL&dl=Pas0Gzpci3aZ41tzhZUllpA1JWtojf3jLxAnbo5zCZg" style="text-decoration: none" title="">Charter</a></p>
		</center>
	</div>

	<div id = 'line_block' ; class="link-ease-in-out">
		<center>
			<p class="a" ><a href="https://vk.com/app5619682_-194667570" style="text-decoration: none" title="">Join</a></p>
		</center>
	</div>


	<div id = 'line_block' ; class="link-ease-in-out">
		<center>
			<p class="a" ><a href="https://vk.com/app5727453_-194667570?ref=group_menu" style="text-decoration: none" title="">Donate</a></p>
		</center>
	</div>


<p><span class="name" ><b>The Commonwealth</b></span></p>


 </body>
</html>

	"""
	return HTMLResponse(content=html_content, status_code=200)
```

### Description

I just started to learn Fast API, first I created a simple `.html` file that works great:

![ok](https://user-images.githubusercontent.com/79650307/234959446-c49a6518-9931-4b09-ad26-9e7f5cda2fcc.png)

BUT when transferring to the framework, the photo is not displayed :(

>![does not work ](https://user-images.githubusercontent.com/79650307/234959832-581903fc-48a6-46f0-ad09-1a6d2011e769.png)

Let me remind you that the `html` code is the same, therefore there is a link to the background is the same:
![image](https://user-images.githubusercontent.com/79650307/234962619-1ca8f541-564c-4eff-b5e4-c098f434150b.png)
![return](https://user-images.githubusercontent.com/79650307/234960269-c308a044-5baf-429e-9f6c-02b496ff07f7.png)
![image](https://user-images.githubusercontent.com/79650307/234961506-c47355ff-334f-4211-831e-caa8d70ea4d5.png)

<hr>

Naturally, I started testing in different ways.
I know how easy it is to upload a picture - and it really shows up!
>![site](https://user-images.githubusercontent.com/79650307/234961070-7e6da972-3a16-432d-8b25-ca6917f4f6fc.png)
>![code](https://user-images.githubusercontent.com/79650307/234961310-d8766b97-2e9e-4670-9937-50d060be9855.png)

By the way, for the first time, the site worked and everything was displayed! I then installed `ngrok` because I wanted to publish the site. I have a suspicion that this is because of him.

### Operating System

Windows

### FastAPI Version

0.95.1

### Python Version

3.10.6

**Comment by jgould22** [comment, marked as answer]
The image file is a "static file"

You can learn more about static files from the docs here https://fastapi.tiangolo.com/tutorial/static-files/

**Reply by IvanIsak2000** [reply]
Thanks a lot!
